src/java-api-bindings/scripts/install_dependencies_and_build.sh

```diff
@@ -98,6 +102,10 @@ for OPTS; do
         export INCLUDE_DEVELOPER_TOOLS_SERVER=0
         echo "Including developer tools server C++ bindings"
         ;;
+        --keep-build-dependencies)
+        KEEP_BUILD_DEPENDENCIES=0
+        echo "Including developer tools server C++ bindings"
```

Comment:
If the below comment gets addressed, we may want to review the other code for things like this which will be inconsistent (e.g. if KEEP=0, then the statement should say we are excluding developer tools bindings).